cpp/include/cudf/unary.hpp

```diff
@@ -31,6 +33,65 @@ namespace cudf {
  * @brief Column APIs for unary ops
  */
 
+/**
```

Comment:
Please add a note to the docs for these functions explaining why we have these conversion functions rather than `explicit` constructors and conversion operators in the fixed-point class.

Reply:
Done